networking_aci/plugins/ml2/drivers/mech_aci/config.py

```diff
@@ -193,6 +199,12 @@ def _parse_fixed_bindings(self):
     def _parse_address_scopes(self):
         self._address_scopes = self._parse_config("address-scope", address_scope_opts, to_dict=True)
 
+        for scope in self._address_scopes:
+            if scope.get('contracts'):
+                contracts = ast.literal_eval(scope.get('contracts'))
+                scope['consumed_contracts'].extend(contracts['consumed'])
+                scope['provided_contracts'].extend(contracts['provided'])
```

Comment:
How bad would it be, if contracts are duplicated in the `consumed_contracts` or `provided_contracts`?

Reply:
We might push them two times to the APIC, but it wouldn't cause a problem